$OpenBSD: patch-schemas_desktop_gnome_interface_schemas_in_in,v 1.1 2009/10/14 15:06:44 ajacoutot Exp $

Display the usual icons under the System menu by default.

--- schemas/desktop_gnome_interface.schemas.in.in.orig	Wed Oct 14 16:49:28 2009
+++ schemas/desktop_gnome_interface.schemas.in.in	Wed Oct 14 16:49:45 2009
@@ -65,7 +65,7 @@
       <applyto>/desktop/gnome/interface/menus_have_icons</applyto>
       <owner>gnome</owner>
       <type>bool</type>
-      <default>false</default>
+      <default>true</default>
       <locale name="C">
         <short>Menus Have Icons</short>
         <long>Whether menus may display an icon next to a menu entry.</long>
